addons/knobs/src/html/index.js

```diff
@@ -0,0 +1,33 @@
+import WrapStory from './WrapStory';
+
+import {
+  knob,
+  text,
+  boolean,
+  number,
+  color,
+  object,
+  array,
+  date,
+  select,
+  files,
+  manager,
+  makeDecorators,
+} from '../base';
+
+export { knob, text, boolean, number, color, object, array, date, select, files };
+
+export function button(name, callback) {
+  return manager.knob(name, { type: 'button', value: Date.now(), callback, hideLabel: true });
+}
+
+function prepareComponent({ getStory, context, channel, knobStore }) {
+  const wrapper = new WrapStory(getStory(context), channel, context, getStory, knobStore);
```

Comment:
You instantiate a new class on each render here, so `connectCallbacks` is a new function each time

Reply:
Yeah, but since the subscriptions store registers it, it holds the reference of every wrapper class. Then on unregister, it removes the previous reference

Reply:
It's not the same reference. So it subscripes to new one and unsubsribes from previous instead of just keeping the previous

Reply:
Actually, I'm not sure there's a use case for a class here

Reply:
> make it similar to other wrappers

No reason for that — this one should eventually replace them all =)